5.37 upgrade notes (#4706)
mattermost/docs: source/administration/important-upgrade-notes.rst

   - In the v5.38 release (August 16, 2021), we will deprecate "config watcher" (the mechanism that automatically reloads the ``config.json file``), in favor of an mmctl command that will need to be run to apply configuration changes after they are made. This change will improve configuration performance and robustness.

+----------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| If you’re upgrading from a version earlier than... | Then...                                                                                                                                                          |
+====================================================+==================================================================================================================================================================+

> | v5.37.0                                            | The ``platform`` binary and “--platform” flag will be deprecated in a future release. If you are using the “--platform” flag or are using the ``platform`` binary|

Reviewer: ```suggestion
| v5.37.0                                            | The ``platform`` binary and “--platform” flag has been removed. If you are using the “--platform” flag or are using the ``platform`` binary|
```

Author: My bad, fixed.

|                                                    | `Collapsed Reply Threads <https://mattermost.com/blog/collapsed-reply-threads-beta/>`_ are available as beta in Mattermost Server                                |
|                                                    | v5.37 and later. It’s expected that you may experience bugs as we stabilize the feature. In particular, please be aware of                                       |
|                                                    | `the known issues documented here <https://docs.mattermost.com/help/messaging/organizing-conversations.html#known-issues>`_.                                     |
|                                                    +------------------------------------------------------------------------------------------------------------------------------------------------------------------+
|                                                    | v5.37 adds support for emoji standard v13.0. If you have added a custom emoji in the past that uses one of the new system names, then it is going to get         |

> |                                                    | overwritten by the system emoji. The workaround is to change the custom emoji name.                                                                              |

Reviewer 2: maybe mention that the emoji picker now supports selecting a default skin tone?

Author: The changelog draft mentions it, I think https://github.com/mattermost/docs/pull/4702/files

Reviewer 2: ah! great then :) disregard my comment
